packages/e2e/features/ui/components/authenticator/sign-in-totp-mfa.feature

```diff
@@ -12,6 +12,8 @@ Feature: Sign In with TOTP MFA
     And I type my password
     And I click the "Sign in" button
     Then I will be redirected to the setup totp page
+    Then I check to see if QR code is correct
+    Then I see the "Confirm" button
```

Comment:
This is well more suited to be tested in a Cypress e2e test then a unit test IMO